code/modules/surgery/organs/augments_chest.dm

```diff
@@ -5,43 +5,6 @@
 	implant_overlay = "chest_implant_overlay"
 	zone = "chest"
 
-/obj/item/organ/cyberimp/chest/nutriment
```

Comment:
So, you're saying, in the future, we still can't make artificial food?

Fuck off.